File: c/01-sequencias/main016.c
/* Escreva um programa para calcular a redução do tempo de vida de um 
fumante. Pergunte a quantidade de cigarros fumados por dias e quantos anos ele 
já fumou. Considere que um fumante perde 10 min de vida a cada cigarro. Calcule 
quantos dias de vida um fumante perderá e exiba o total em dias.

Data 29/09/2025 -> 8ª Dia
*/

#include <stdio.h>

int main() {

    printf("=== CALCULADOR DE TEMPO DE VIDA PERDIDO ===\n");

    int cigarro_por_dia;
    int anos_fumado;
    int total_cigarro;
    int minutos_perdidos;
    float dias_perdidos;

    printf("Quantos cigarros fuma por dia? ");
    scanf("%d",&cigarro_por_dia);

    printf("Ha quanto anos fuma? ");
    scanf("%d",&anos_fumado);

    total_cigarro = cigarro_por_dia * 365 * anos_fumado;
    minutos_perdidos = total_cigarro * 10;
    dias_perdidos = minutos_perdidos / (24*60);

    printf("--- RESULTADO ---\n");
    printf("Cigarros fumados na vida: %d\n",total_cigarro);
    printf("Minutos de vida perdidos: %d\n",minutos_perdidos);
    printf("Dias de vida perdidos: %.1f\n",dias_perdidos);
    printf("Voce perdeu aproximadamente %.0f dias de vida",dias_perdidos);
    return 0;
    
}

File: python/01-sequencias/main016.py
'''Escreva um programa para calcular a redução do tempo de vida de um 
fumante. Pergunte a quantidade de cigarros fumados por dias e quantos anos ele 
já fumou. Considere que um fumante perde 10 min de vida a cada cigarro. Calcule 
quantos dias de vida um fumante perderá e exiba o total em dias.

Data 29/09/2025 -> 8ª Dia
'''

print("=== CALCULADORA DE TEMPO DE VIDA PERDIDO ===")

cigarro_por_dia = int(input('Quantos cigarros fuma por dia? '))
anos_fumado = int(input('Há quantos anos fuma? '))

total_cigarro = cigarro_por_dia * 365 * anos_fumado
minutos_perdidos = total_cigarro * 10
dias_perdidos = minutos_perdidos / (24*60)

print('--- RESULTADO ---')
print(f'Cigarros fumados na vida: {total_cigarro}')
print(f'Minutos de vida perdidos: {minutos_perdidos}')
print(f'Dias de vida perdidos: {dias_perdidos:.1f}')
print(f'Você perdeu aproximadamente {dias_perdidos:.0f} dias de vida!')